### PR TITLE
fix:fix duplicate import of flow library #98

### PR DESCRIPTION
**Pull request by Dibyanshuaman**

### CODE REVIEW

File: pkg/module/metrics/latency.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 	"time"
 
 	"github.com/cilium/cilium/api/v1/flow"
-	v1 "github.com/cilium/cilium/api/v1/flow"
+
```
Comment (rbtr) [inline pkg/module/metrics/latency.go:15]: the linter will not like this empty line, it should be removed

**Review by rbtr — CHANGES_REQUESTED** [review]

**Comment by rbtr** [comment]
thanks for the contribution. please amend your commit message so that it is in [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) format, and add "fixes #93" to this PR description to link back to the issue that this fixes

File: pkg/module/metrics/latency.go (hunk 1 above)
Comment (vakalapa) [reply pkg/module/metrics/latency.go:15]: Also there are references of this v1 package name which needs to be corrected as well.

**Comment by FZhg** [comment]
> thanks for the contribution. please amend your commit message so that it is in [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) format, and add "fixes #93" to this PR description to link back to the issue that this fixes

Hi @rbtr, it should fix #98

File: pkg/module/metrics/latency.go [hunk 2]
```diff
@@ -11,7 +11,6 @@ import (
 	"sync"
 	"time"
 
-	"github.com/cilium/cilium/api/v1/flow"
```
Comment (anubhabMajumdar) [inline pkg/module/metrics/latency.go:14]: I believe there are other references to `flow`, example - https://github.com/microsoft/retina/blob/96ff934a608fcce848d66c31ad8e54afbed9e00e/pkg/module/metrics/latency.go#L60

**Review by anubhabMajumdar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**